ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -1209,11 +1214,14 @@ def main():
         supports_check_mode=False)
 
     cmd = module.params['cmd']
+    vm_set_name = module.params['vm_set_name']
```

Comment:
Not all ops have `vm_set_name` defined, could you please verify?

Reply:
Thanks for the remider, and I noticed that, here's the sample result, 'destroy' and 'create' don't have vm_set_name :

/tmp$ ls|grep vm_topo
vm_topology_2021-11-29T03:16:36.255083.log (old)
vm_topology_2021-11-29T03:17:03.194976.log (old)
vm_topology_bind_vms20-2_2021-11-29T07:44:15.935972.log (new)
vm_topology_create_2021-11-29T07:43:52.758181.log (new)
vm_topology_destroy_2021-11-29T07:40:58.813069.log (new)
vm_topology_renumber_vms20-2_2021-11-29T08:09:14.360649.log (new)
vm_topology_unbind_vms20-2_2021-11-29T07:40:28.707791.log (new)